timeline/templatetags/timeline_tags.py

```diff
@@ -37,6 +37,13 @@ def get_timestamp(event):
     return localtime(event.get_timestamp()).strftime('%Y-%m-%d %H:%M:%S')
 
 
+@register.simple_tag
```

Comment:
A template tag is not needed, you can simply call `user.is_superuser` in a template.

Reply:
I reinvented a bicycle :) Fixed